CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.5.13] - 2018-12-13
+### Fixed
+- Se corrige un problema relacionado a 'onepayCountdown' que al expirar el tiempo asignado para terminar la transaccion por parte del usuario, retorna 'updateContentError' con estado 'undefined' no respetando idempotencia de la transaccion, se modifica para que retorne el estado
```

Comment:
The changelog should explain things for the user of the SDK and not for the implementor of the SDK. Rather than explaining what was changed, explain *why* it had to be changed (i.e, what effect it has or what problem it fixes but without going into *how* it was fixed)